src/DOM/wrappers/TextareaWrapper.ts

```diff
@@ -70,7 +70,7 @@ export function applyValue(vNode, dom) {
 	const value = props.value;
 	const domValue = dom.value;
 
-	if (domValue !== value) {
+	if (!isNullOrUndef(value) && domValue !== value) {
```

Comment:
Maybe small refactoring needed to this function as its doing the same check just below? - Reorder it so we check only once?